include/splitvector/splitvec.h

```diff
@@ -421,26 +428,31 @@ class SplitVector {
 
       _deallocate_and_destroy(capacity(), _data);
       _data = other._data;
-      *_size = other.size();
-      *_capacity = other.capacity();
-      *(other._capacity) = 0;
-      *(other._size) = 0;
+      _info->size = other.size();
+      _info->capacity= other.capacity();
+      other._info->capacity = 0;
+      other._info->size = 0;
       other._data = nullptr;
       _location = other._location;
```

Comment:
`_allocator = other._allocator` as well?

Reply:
The assignment operator should not copy over the allocator

Reply:
I think I disagree. The reason is this: the operation steals the _data address from the other vector. On exit, the splitvector will try to deallocate that memory, but that allocator won't know of that memory address!


And to be complete, this should thus also steal the `_info` allocation from the other vector (or use the new allocator to reallocate one).